core/server/common/src/main/java/alluxio/RpcUtils.java

```diff
@@ -39,10 +39,35 @@
     try {
       return callable.call();
     } catch (AlluxioException e) {
-      LOG.debug("Internal Alluxio error when running rpc", e);
+      LOG.debug("Internal Error: {}", callable, e);
```

Comment:
"Internal Error" suggests that something has gone wrong in the server, but this may not be the case, e.g. someone tries to create a file but it already exists. I'd prefer something more neutral like `Exception thrown by {}`.